packages/vats/src/core/types.js

```diff
@@ -67,21 +95,21 @@
  *   assignBundle: (ps: PropertyMakers) => void
  * }} ClientManager
  *
- * @typedef {Record<string, (addr: string) => void>} PropertyMakers
+ * @typedef {Array<(addr: string) => Record<string, unknown>>} PropertyMakers
  */
 
 /**
  * @callback CreateUserBundle
  * @param {string} nickname
  * @param {string} clientAddress
  * @param {string[]} powerFlags
- * @returns {Promise<Record<string, unknown>>}
+ * @returns {Promise<Record<string, Promise<any>>>}
```

Comment:
This looks like a regression in type safety. How hard would it be to keep the `unknown` instead of `any`?

Reply:
_this justification belongs in the code..._

The userBundle really is an arbitrary pile of properties. And it's not like you can check them with `typeof`. You are either in a position to trust them or you're not. With `unknown`, each client that wants to call a method such as `E(userBundle.bank).deposit(payment)` has to cast `userBundle.bank`; ideally, the cast is to some useful type. But `unknown` can't be cast directly to some other type; it has to be cast to `any` first. I'm not sure all that hassle is cost-effective.